### ISSUE TITLE
Method to set dask config path

### ISSUE THREAD

**Issue by mangecoeur** [dask/distributed#744]
I'm using dask distributed on an HPC cluster that makes $HOME not writable from batch jobs (can only write to a Scratch space). Ran into failures where dask tried to initialize its config from a job. The solution is to simply change where dask stores its config, currently it's hardcoded to the user home. IPython parallel and jupyter let you set environment variables to override the defaults which seems to work fine.

**Comment by mrocklin** [comment]
That seems reasonable to me.  Do you have any desire to write up a PR that
mimics the solution used by IPyParallel or Jupyter?

**Comment by pitrou** [comment]
I agree the expected idiom would be to provide an environment variable to override the config file location. For example git has `GIT_CONFIG` and Mercurial has `HGRCPATH`.
